README.md

```diff
@@ -4,9 +4,12 @@
 | --- | --- | --- | --- |
 | [![Build Status][Badge-Circle]][Link-Circle] | [![Release Artifacts][Badge-SonatypeReleases]][Link-SonatypeReleases] | [![Snapshot Artifacts][Badge-SonatypeSnapshots]][Link-SonatypeSnapshots] | [![Badge-Discord]][Link-Discord] |
 
-ZIO-based cluster membership and failure detection.
+ZIO-based liabrary for cluster membership and failure detection using gossip based protocol.
+
+This library is inspired by Hashicorp research on improving [SWIM](https://ieeexplore.ieee.org/document/1028914) protocol published in [Lifeguard](https://ieeexplore.ieee.org/document/1028914) paper. 
+
+Please find more details on our [website](https://zio.github.io/zio-memberlist/).
 
-> This library is still a work-in-progress. Watch this space!
 
 # License
 [License](LICENSE)
```

Comment:
Please add apache2 license as per other projects

Reply:
what do you mean? project was created from template and Apache 2.0 is in it https://github.com/zio/zio-memberlist/blob/master/LICENSE

Reply:
AH yes my mistake! Sorry about that.